### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by sudarsan-surendralal**

### CODE REVIEW

**Comment by sudarsan-surendralal** [comment]
@yury-lysogorskiy, I have used your snippets to interface with your structure database in this notebook. Can you have a look?

File: day_3/ex_01_validation.ipynb [hunk 1]
```diff
@@ -0,0 +1,2247 @@
+{
```
Comment (yury-lysogorskiy) [inline day_3/ex_01_validation.ipynb:1]: <p>could you print group_name before each plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron_potentialfit/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (yury-lysogorskiy) [inline day_3/ex_01_validation.ipynb:1]: <p>Why C-constants are identical for both potentials? And it also looks too high wrt. DFT values...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron_potentialfit/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (yury-lysogorskiy) [inline day_3/ex_01_validation.ipynb:1]: <p>Cu-ace (forces) and Cu-runner (forces) looks identical. Something wrong...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron_potentialfit/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

**Comment by jan-janssen** [comment]
@mrshrbld Updated the runner potentials in https://github.com/pyiron/pyiron_potentialfit/pull/47/ - they are now named: `Cu-runner-df4-lowerRMSE`, `Cu-runner-df4` and `Cu-runner-df1`.

File: day_3/ex_01_validation.ipynb (hunk 1 above)
Comment (sudarsan-surendralal) [reply day_3/ex_01_validation.ipynb:1]: Thanks this is now fixed!

Comment (sudarsan-surendralal) [reply day_3/ex_01_validation.ipynb:1]: Yeah, my mistake. This is fixed!

**Comment by jan-janssen** [comment]
```
---------------------------------------------------------------------------
358
Exception encountered at "In [8]":
359
---------------------------------------------------------------------------
360
NameError                                 Traceback (most recent call last)
361
<ipython-input-8-fd05abbf45b3> in <module>
362
----> 1 murn_job["output/equilibrium_energy"]
363

364
NameError: name 'murn_job' is not defined
```
@sudarsan-surendralal Is there a dependency of the notebooks?

**Comment by sudarsan-surendralal** [comment]
> 
> 
> ```
> ---------------------------------------------------------------------------
> 358
> Exception encountered at "In [8]":
> 359
> ---------------------------------------------------------------------------
> 360
> NameError                                 Traceback (most recent call last)
> 361
> <ipython-input-8-fd05abbf45b3> in <module>
> 362
> ----> 1 murn_job["output/equilibrium_energy"]
> 363
> 
> 364
> NameError: name 'murn_job' is not defined
> ```
> 
> @sudarsan-surendralal Is there a dependency of the notebooks?

No. Could it be because I use `%%time` in the previous cell? (https://github.com/ipython/ipython/issues/11659/) I've removed this

**Comment by jan-janssen** [comment]
The potential name `Cu_Mendelev_eam` is outdated it should be `2012--Mendelev-M-I--Cu--LAMMPS--ipr1` following the NIST naming conventions.

**Pull request merged**